app/models/form526_submission.rb

```diff
@@ -566,4 +566,47 @@ def poll_form526_pdf
   def cleanup
     EVSS::DisabilityCompensationForm::SubmitForm526Cleanup.perform_async(id)
   end
+
+  def account
+    # first, check for an ICN on the UserAccount associated to the submission, return it if found
+    account = user_account
+    return account if account.present? && account.icn.present?
+
+    # next, check past submissions for different UserAccounts that might have ICNs
+    past_submissions = get_past_submissions
+    account = find_user_account_with_icn(past_submissions, 'past submissions')
+    return account if account.present? && account.icn.present?
```

Comment:
Because checking `account.present?` already implies that `account` is not `nil`, thus `account.icn` can be checked directly, simplified with a safe navigation operator as `account&.icn.present?`